conda_smithy/templates/appveyor.yml.tmpl

```diff
@@ -69,4 +69,4 @@ deploy_script:
 {%- if upload_on_branch %}
     - set "UPLOAD_ON_BRANCH={{ upload_on_branch }}"
 {%- endif %}
-    - cmd: upload_package {% if conda_forge_output_validation %}--validate --feedstock-name="{{ feedstock_name }}"{% endif %} .\ .\{{ recipe_dir }} .ci_support\%CONFIG%.yaml
+    - cmd: upload_package {% if conda_forge_output_validation %}--validate --feedstock-name="{{ feedstock_name }}"{% endif %} {% if conda_forge_private_upload %}--private {% endif %} .\ .\{{ recipe_dir }} .ci_support\%CONFIG%.yaml
```

Comment:
This line will introduce an extra space in the rendered configs which will cause extra commits to feedstocks. There is a space after `--private` and one between the two if blocks. Let's remove those.

Reply:
Done. Are you sure about removing the space between the two blocks?

```
{{ feedstock_name }}"{% endif %}{% if conda_forge_private_upload %}
```

Reply:
Right. Add an extra space before `--private` too